test/helpers/deployTerms.js

```diff
@@ -49,7 +52,7 @@ export const defEtoTerms = {
   ENABLE_TRANSFERS_ON_SUCCESS: false,
   INVESTOR_OFFERING_DOCUMENT_URL: "893289290300923809jdkljoi3",
   SHAREHOLDER_RIGHTS: null,
-  WHITELIST_DISCOUNT_FRAC: Q18.mul(0.3),
+  WHITELIST_DISCOUNT_FRAC: Q18.mul("0.3"),
```

Comment:
do we need the quotes for it to work? If so, than there a bunch of other decimal numbers in line 15-22 for ```defaultShareholderTerms``` which need to be quoted too!

Reply:
yes we should. but there's some trickery happening with number literals in javascript. there's no finite binary representation of 0.3 but still we obtain correct big number... I will fix wherever I can

btw. in Python there's no mercy, try this
```
>>> Decimal(0.3)
Decimal('0.299999999999999988897769753748434595763683319091796875')
```